src/test/java/org/gitlab4j/api/TestProjectApi.java

```diff
@@ -224,6 +224,7 @@ public void testCreate() throws GitLabApiException {
                 .withMergeRequestsEnabled(true)
                 .withWikiEnabled(true)
                 .withSnippetsEnabled(true)
+                .withRepositoryObjectFormat("sha1")
```

Comment:
If I am not mistaken this test is using the GitLab installation started in Docker while the integration tests are running.

This version is very old and this test will fail.

Reply:
Hmm... Is there any reference test that I could use for this new field?

Reply:
The pipeline in GitHub action is failing as expected:
```
Error:  Failures: 
Error:    TestProjectApi.testCreate:239 expected: <sha1> but was: <null>
[INFO] 
Error:  Tests run: 308, Failures: 1, Errors: 0, Skipped: 14
```

Reply:
Some tests are using Mockito but this feels also wrong.
I started to experiment to use testcontainers to run a newer GitLab version (there is some work in a PR).

But I am wondering if we should not just Mock the GitLab server in order to run the unit-test quickly.

I would say that for now `TestProjectApi.testCreate()` remains unchanged and we merge your change.